server/docker-compose.yml

```diff
@@ -23,17 +22,18 @@ services:
       args:
         IMPORT_TOOL_PATH: ""
         C3_URL: https://c3_url
-        DB_URL: sqlite:////home/code/test.db
+        DB_URL: sqlite:////home/server/test.db
     command: >
       bash -c '
         black --check hwapi/ scripts/ tests/ &&
         ruff check hwapi/ scripts/ tests/ --no-cache &&
         mypy hwapi/ scripts/ tests/ &&
-        coverage run -m pytest tests/ -vv &&
-        coverage xml -o coverage/coverage.xml'
-    working_dir: /home/code
+        coverage run --source "hwapi/" -m pytest tests/ -vv &&
+        cd .. && cp server/.coverage . &&
```

Comment:
this change was made according to this solution for fixing coverage report path: https://github.com/pytest-dev/pytest-cov/issues/305#issuecomment-508691008